### ISSUE TITLE
Evergreen installer doesn't work when launched from Advanced Installer

### ISSUE THREAD

**Issue by Saurbaum** [MicrosoftEdge/WebView2Feedback#1582]
I've configured my advanced installer project to run the WebView2 evergreen runtime and the installation fails with error code 0x80040c01. Offline machine as I have to support no internet access on the final product installation.

![image](https://user-images.githubusercontent.com/7288848/127493003-e9b76386-81bf-4533-b73d-c50fa92eb519.png)

If I run the same exe outside of my installer it works but I'm at a loss to explain why this isn't working when triggered during my install.

Can I get an idea what this error is caused by?

**Comment by Saurbaum** [comment]
If I run the same package manually it only works on Windows 10, as soon as I try it on server 2019 I get this 
![image](https://user-images.githubusercontent.com/7288848/127664620-2886eb10-fbd1-41a2-a10f-c364e859db45.png)

**Comment by Saurbaum** [comment]
Did someone update the evergreen installer without changing the version numbers in the last week. I downloaded it again to compare to the version I had before I see this

![image](https://user-images.githubusercontent.com/7288848/127665972-c049df4e-033a-4b4d-8992-1681eecc296c.png)

**Comment by champnic** [comment]
Hey @Saurbaum - are you running the installer elevated?

**Comment by champnic** [comment]
It looks like the error code `0x80040c01` can be due to incorrect parameters when calling the installer. What is the command-line of the failed installer (your first image)?

**Comment by Saurbaum** [comment]
I tried with /silent /install and I also tried with no parameters

**Comment by Saurbaum** [comment]
Yes the installation is elevated

**Comment by champnic** [comment]
Can you verify by looking at the process that you are getting the expected command line (and that Advanced Installer isn't adding anything, for example) in the Task Manager > Details tab > Command Line column.

**Comment by Saurbaum** [comment]
Yes the same parameters are being passed through.

**Comment by Saurbaum** [comment]
I've noticed that as part of our build we digitally sign everything so we know it's the same as when it left us. This includes 3rd party dependencies.

Are you doing something at install time like checking your installer has been signed by a specific signature?

**Comment by champnic** [comment]
That's possible - let me check with the installer team.

**Comment by champnic** [comment]
Hey @Saurbaum - it sounds like we do have signature verification for our installer. Are you able to share your built installer so we can verify? If you feel more comfortable you can email it to me at `champnic@microsoft.com`. You can also upload it and send me a link if that works for you. Thanks!

**Comment by Saurbaum** [comment]
I have currently stopped signing the runtime and it's working. Seems that if you resign it at all the installer stops which is unlike any of the other runtimes (like VC++) we distribute.

**Comment by champnic** [comment]
Sounds good - glad you've got it working! FWIW we think that adding a signature (-as) should work, but that overwriting a signature chain would cause this issue. Thanks!

**Comment by anuragmuley09** [comment]
I'm trying to run the Tauri application the log says exe running but I can't see the Tauri app window. 
Few days back I cleaned all the files from %TEMP% (via win+r), is that the issue?

I suspected my core c++ - rust code before pointing eyes towards angular or Tauri setup, but everything seems to be just fine. Then later I created a new separate Tauri application, and again same results, terminal says exe running but I can't see it, and neither in Task Manager. 

So my doubt is on MicrosoftEdge WebView2. 
I tried installing evergreen installer, downloaded it from the official source

<img width="845" height="417" alt="Image" src="https://github.com/user-attachments/assets/90da27c0-af50-419a-a017-182204bc72f6" />

<img width="1241" height="629" alt="Image" src="https://github.com/user-attachments/assets/31e4a962-eab2-4fe1-9211-bb809161c767" />

Now, i dont really know whats happening, WebView isnt updating itself neither allowing installer to do its stuff. 

Your response is appreciated! Thanks!
